src/components/BaseSelectValidation.vue

```diff
@@ -53,3 +61,40 @@ export default {
   }
 }
 </script>
+<style src="styles/spacing.scss" lang="scss" module="space"></style>
+
+<style lang="scss" module="base">
+@import '~styleConfig/color';
+
+.input {
+  composes: default from 'styles/animation.scss';
+  composes: paddingXnarrow from 'styles/spacing.scss';
+  composes: round default from 'styles/borders.scss';
+  composes: lightBg from 'styles/color.scss';
+  box-shadow: none !important;
+  display: block;
+  width: 100%;
+  outline: 0;
+  outline: thin dotted \9;
+
+  &:focus,
+  &:active {
+    border-color: color('highlight');
+    background-color: color('white');
+  }
+}
+
+.textarea {
+  composes: input;
+  height: auto;
+  min-height: 4em;
+  resize: vertical;
+}
+
+.error {
+  composes: marginTopXnarrow noMarginBottom paddingXnarrow from 'styles/spacing.scss';
+  composes: default round from 'styles/borders.scss';
+  composes: scaleZeta from 'styles/type.scss';
+  composes: warningBorder warning from 'styles/color.scss';
+}
+</style>
```

Comment:
Is this a copy of the styles from `BaseFormInput`? If so I'll make a shared stylesheet.